dolphinscheduler-task-plugin/dolphinscheduler-task-zeppelin/pom.xml

```diff
@@ -43,5 +43,23 @@
             <artifactId>zeppelin-client</artifactId>
             <version>0.10.1</version>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>3.9.0</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-junit-jupiter</artifactId>
+            <version>3.9.0</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-inline</artifactId>
+            <version>3.9.0</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Is it better to put this outside the `task-plugin`?

Reply:
Certainly yes.

Reply:
@SbloodyS @kezhenxu94 Looks like putting the dependencies in the root pom.xml will cause quite many UTs which use `Powermock` to fail. What about doing it module by module? Since this PR is just an example for removing `Powermock`, I will put these dependencies in `dolphinscheduler-task-plugin/pom.xml` and remove all the code related to `Powermock` in `dolphinscheduler-task-plugin` module.